pages/science/species-thermo.rst

```diff
@@ -80,25 +80,14 @@ expressions to compute the properties, they all require thermodynamic property
 information for the individual species. For the phase types implemented at
 present, the properties needed are:
 
-1. the molar heat capacity at constant pressure :math:`\hat{c}^0_p(T)` for a
```

Comment:
There's a link a few lines up from here which is broken now, it should be `/science/phase-thermo.html` I think.